src/simsopt/_core/finite_difference.py

```diff
@@ -209,6 +211,9 @@ def _jac(self, x: RealArray = None):
         nparams = opt.dof_size
         # Make sure all leaders have the same x0.
         mpi.comm_leaders.Bcast(x0)
+        if not args == (): non_dofs = np.array(args)
+        else: non_dofs = None
```

Comment:
Not pythonic. Use 
`non_dofs = np.array(args) if args else None`

Reply:
Fixed in commit https://github.com/hiddenSymmetries/simsopt/pull/301/commits/e68266e3a0da0392ad541225d835a9eace25122d